src/common/src/config.rs

```diff
@@ -526,6 +526,10 @@ pub struct StreamingConfig {
     #[serde(default = "default::streaming::unique_user_stream_errors")]
     pub unique_user_stream_errors: usize,
 
+    /// Control the strictness of stream consistency.
+    #[serde(default = "default::streaming::unsafe_enable_strict_consistency")]
+    pub unsafe_enable_strict_consistency: bool,
```

Comment:
Just suddenly come to my mind: should we make it `unsafe_disable` instead? 🤣 Otherwise it sounds like "enabling this option is unsafe".

Reply:
I think `unsafe_` means it's unsafe to touch this config? I personally prefer all boolean config items to use `enable_` instead of `disable_` in their name.

Reply:
Actually the best way may be the old solution: `unsafe_strict_consistency: enum { On, Off }`. That's more clearer.